### DISCUSSION TITLE
Is there any way to turn a field into an editable inline?

### DISCUSSION THREAD

**Discussion by didix16** [orchidsoftware/platform#1744] [Q&A]
So as the title says, is there any way (for example, in a resource list Screen like a list of users) to render some kind of input field that can be updated inline? For example, let's say we have a User with an "enabled" boolean field. You just want to toggle its status with the switcher and that this status change is shown on the database automatically.

this could be an example:
![image](https://user-images.githubusercontent.com/10291092/119495741-03513e00-bd63-11eb-97fa-78b376ab2ace.png)

If I change the published field switcher, then in the database I should see a falsy value.

BTW: It would be nice to apply to any kind of fields like inputs or date/times, numbers, etc

**Comment by mabdelfattah** [comment]
2 years later, and I have the same question. :) I want to allow toggle inline instead of opening a new modal just for toggle! And the same for select.
